src/databricks/labs/ucx/contexts/workspace_cli.py

```diff
@@ -170,3 +167,22 @@ def iam_role_creation(self):
             self.workspace_client,
             self.aws_resource_permissions,
         )
+
+    @cached_property
+    def notebook_loader(self) -> NotebookLoader:
+        return NotebookLoader()
+
+
+class LocalContext(WorkspaceContext):
```

Comment:
```suggestion
class LocalCheckoutContext(WorkspaceContext):
```
`local` is too ambiguous here

Reply:
done